corehq/apps/userreports/mixins.py

```diff
@@ -82,9 +82,8 @@ def set_filter_values(self, filter_values):
         for filter_slug, value in filter_values.items():
             self._filter_values[filter_slug] = self._filters[filter_slug].create_filter_value(value)
 
-    def defer_filters(self, filter_slugs):
-        self._deferred_filters.update({
-            filter_slug: self._filters[filter_slug] for filter_slug in filter_slugs})
+    def set_defer_fields(self, defer_fields):
+        self._defer_fields = defer_fields
```

Comment:
I think that `deferred_filters` was a better name for this since it's saying filters that are deferred until they reach the phone right?

Reply:
It's the just the objects that are being passed in here are not full filter objects but just field names, so I renamed it so.